tests/store/artifact/test_cli.py

```diff
@@ -71,21 +82,61 @@ def test_get_artifact_repository(artifact_uri):
             assert expected_result == actual_result
 
 
-def test_download_artifacts_from_uri():
-    with mlflow.start_run() as run:
-        with TempDir() as tmp:
-            local_path = tmp.path("test")
-            with open(local_path, "w") as f:
-                f.write("test")
-            mlflow.log_artifact(local_path, "test")
-    command = ["mlflow", "artifacts", "download", "-u"]
-    # Test with run uri
-    run_uri = "runs:/{run_id}/test".format(run_id=run.info.run_id)
-    actual_uri = posixpath.join(run.info.artifact_uri, "test")
+def _run_download_artifact_command(command):
+    """
+    :param command: An `mlflow artifacts` command list.
+    :return: The downloaded artifact content.
+    """
+    p = Popen(command, stdout=PIPE, stderr=STDOUT)
+    output = p.stdout.readlines()
+    download_output_path = pathlib.Path(output[-1].strip().decode("utf-8"))
+    downloaded_file = list(download_output_path.iterdir())[0]
```

Comment:
```suggestion
    downloaded_file = next(download_output_path.iterdir())
```

nit: Is `next` simpler than `ilst(...)[0]`?

Reply:
Heck yeah! Done!